### PR TITLE
The issue #83 was addressed

### PR DESCRIPTION
**Pull request by SNikalaichyk**
The **Confirm-UniqueBinding** helper function was updated to address issue #83. Unit tests updated to reflect changed functionality.
The **Set-TargetResource** function: website creation code was reorganized for better error handling.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Currently, only FastCgiModule is supported.
 * **Name**: The desired name of the website.
 * **PhysicalPath**: The path to the files that compose the website.
 * **State**: The state of the website: { Started | Stopped }
-* **BindingInfo**: Website's binding information in the form of an array of embedded instances of the MSFT_xWebBindingInformation CIM class.
+* **BindingInfo**: Website's binding information in the form of an array of embedded instances of the **MSFT_xWebBindingInformation** CIM class.
 * **ApplicationPool**: The website’s application pool.
 * **EnabledProtocols**: The protocols that are enabled for the website. 
 * **Ensure**: Ensures that the website is **Present** or **Absent**.
```
Comment (TravisEz13) [inline README.md]: Please include a summary of changes in an unreleased version change per the [contribution guidelines](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#lifecycle-of-a-pull-request)

**Comment by TravisEz13** [comment]
LGTM if my one comment is addressed.

**Comment by SNikalaichyk** [comment]
**README.MD** updated:
* Added description of the addressed issue in the **Unreleased** section.
* Added detailed description for all of the **MSFT_xWebBindingInformation** properties that are currently supported.

**Comment by tysonjhayes** [comment]
LGTM. Thanks @SNikalaichyk

**Pull request merged**